src/Test/SchemaResetterInterface.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Zenstruck\Foundry\Test;
+
+/**
+ * @internal
+ */
+interface SchemaResetterInterface
```

Comment:
Since this is an internal interface, let's just make it an abstract class (`SchemaResetter`) that includes the helper method from the `CommandRunner` trait and remove the trait.